streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -1246,7 +1246,7 @@ public void cleanUp() {
         }
 
         log.debug("Current changelog positions: {}", allChangelogPositions);
-        final Map<TopicPartition, ListOffsetsResultInfo> allEndOffsets = fetchEndOffsetsWithoutTimeout(allPartitions, adminClient);
+        final Map<TopicPartition, ListOffsetsResultInfo> allEndOffsets = fetchEndOffsets(allPartitions, adminClient);
```

Comment:
Can we please update the JavaDocs of `allLocalStorePartitionLags` to state that a timeout exception could be thrown?

Reply:
ack, done